bin/varnishd/mgt/mgt_param_tweak.c

```diff
@@ -67,29 +69,29 @@ parse_decimal(const char *p, const char **err)
 
 static int
 tweak_generic_double(struct vsb *vsb, const char *arg, const struct parspec *pp,
-    const char *fmt)
+    parse_double_f parse, const char *fmt)
```

Comment:
I would have made the parse argument default to parse_decimal if NULL.

Reply:
Do we really need a default for a static function with 3 call sites?